### ISSUE TITLE
ViroCamera's rotation cannot be set via setNativeProps

### ISSUE THREAD

**Issue by meatwallace** [viromedia/viro#396]
- [x] Review the documentation: https://docs.viromedia.com/
  - [x] Search for existing issues: https://github.com/viromedia/viro
  - [x] Use the latest ViroReact release: https://docs.viromedia.com/docs/releases

## Environment

Please provide the following information about your environment:

1. Development OS: Mac
2. Device OS & Version: Android 7.x
3. Version: `viro-react@2.9.1` & `react-native@0.55.1`
4. Device(s): Samsung S6/S7

## Description

Attempting to set a `ViroCamera`'s `rotation` via the instance's `setNativeProps` method has no effect. Setting `position` works as expected.

Discovered this while trying to implement touch panning as described in [this issue](#103). Rotating the `Viro360Image` works as expected, however is an undesirable approach due to everything else in the scene remaining fixed.

## Reproducible Demo

```js
export class ExampleScene extends React.Component<Props> {
  camera = React.createRef();

  componentDidMount() {
    setInterval(() => {
      this.camera.current.setNativeProps({
        rotation: [
          Math.random() * 360,
          Math.random() * 360,
          Math.random() * 360
        ]
      });
    }, 5000);
  }

  render() {
    return (
      <ViroScene>
          <ViroCamera ref={this.camera} active />
      </ViroScene>
    );
  }
}
```

**Comment by achuvm** [comment]
Hi @meatwallace,

This looks like a bug in our platform. It seems as if the `rotation` prop isn't work at all.

We'll try to get this in our next release.

Thanks,

**Comment by dam00n** [comment]
This has been fixed in ViroReact 2.10. Release notes and upgrade instructions here -> https://docs.viromedia.com/docs/releases

Please re-open the issue if you are still having problems after upgrading.
